compiler-plugin/src/main/java/io/ballerina/stdlib/sql/compiler/Constants.java

```diff
@@ -96,6 +96,15 @@ public static class OutParameter {
         public static final String XML = "XMLOutParameter";
     }
 
-    public static final String UNNECESSARY_CHARS_REGEX = "\"|\\n";
+    /**
+     * Constants for Ballerina time module record types.
+     */
+    public static class TimeRecordTypes {
+        public static final String TIME_OF_DAY = "TimeOfDay";
+        public static final String DATE = "Date";
+        public static final String CIVIL = "Civil";
+        public static final String UTC = "Utc";
```

Comment:
Aren't we validating the package name. This will pass for say civil record from a custom package.

Reply:
Added regex validation with package name in 
880af3f